qpdf-sys/build.rs

```diff
@@ -166,6 +186,7 @@ fn base_build() -> cc::Build {
     build
         .warnings(false)
         .extra_warnings(false)
+        .define("POINTERHOLDER_TRANSITION", "0")
```

Comment:
windows will fail because of that, because PointerHolder.hh uses #warning pragma which is not supported on cl.exe